jobs/build/ocp/Jenkinsfile

```diff
@@ -370,6 +374,25 @@ node(TARGET_NODE) {
             }
         }
 
+        stage( "prep web-console-server" ) {
```

Comment:
Is the stage that generates the console vendor commit after this one.

Reply:
Yes. https://github.com/openshift/aos-cd-jobs/pull/993/files#diff-4afa940aacab03ebeba0c98c8cc67d64R434